sdk/monitor/Azure.Monitor.OpenTelemetry.Exporter/src/Internals/InnerExceptionCountExceededException.cs

```diff
@@ -10,7 +10,6 @@ namespace Azure.Monitor.OpenTelemetry.Exporter.Internals
     /// <summary>
     /// This exception is used to notify the user that the set of inner exceptions has been trimmed because it exceeded our allowed send limit.
     /// </summary>
-    [Serializable]
```

Comment:
See: [SYSLIB0051: Legacy serialization support APIs are obsolete](https://learn.microsoft.com/en-us/dotnet/fundamentals/syslib-diagnostics/syslib0051)

Reply:
I would love for us to just be able to remove this annotation and the protected constructor ... but is it a breaking change we are allowed to take?  If it is, it would be great to do it in Azure.Core too.

Reply:
This one is internal so it should be fine to remove. We could probably take the breaking change, but the guidance on the warning is to mark it as obsolete in cases where there are API backwards-compatibility requirements.